deps/exokit-bindings/webglcontext/src/webgl.cc

```diff
@@ -4124,27 +4125,27 @@ NAN_METHOD(WebGLRenderingContext::GetShaderSource) {
 }
 
 NAN_METHOD(WebGLRenderingContext::ValidateProgram) {
-  GLuint programId = info[0]->ToObject()->Get(JS_STR("id"))->Uint32Value();
+  GLuint programId = TO_UINT32(JS_OBJ(info[0])->Get(JS_STR("id")));
 
   glValidateProgram(programId);
 }
 
 NAN_METHOD(WebGLRenderingContext::TexSubImage2D) {
   WebGLRenderingContext *gl = ObjectWrap::Unwrap<WebGLRenderingContext>(info.This());
-  GLenum targetV = info[0]->Uint32Value();
-  GLint levelV = info[1]->Int32Value();
-  GLint xoffsetV = info[2]->Int32Value();
-  GLint yoffsetV = info[3]->Int32Value();
-  GLsizei widthV = info[4]->Uint32Value();
-  GLsizei heightV = info[5]->Uint32Value();
-  GLenum formatV = info[6]->Uint32Value();
-  GLenum typeV = info[7]->Uint32Value();
+  GLenum targetV = TO_UINT32(info[0]);
```

Comment:
should this be `TO_INT32` for casting to `GLenum`?

Reply:
Yup.